src/main/java/tasks/DukeTask.java

```diff
@@ -0,0 +1,78 @@
+package tasks;
+
+public class DukeTask {
+    protected final String name;
+    protected final boolean isDone;
```

Comment:
It is good that your boolean variables are named to sound like booleans. I also noticed the same practice in several places such as boolean method names too. Keep it up!